console-subscriber/src/builder.rs

```diff
@@ -138,7 +140,7 @@ impl Builder {
     /// defaults.
```

Comment:
this documentation should be updated to state that the provided address can be a TCP address, or (on Unix systems) a UDS address. It would be nice to show examples of both usages in this method's documentation.

Reply:
Done.